### PR TITLE
feat: issue-5712 add sleep tool

### PR DESCRIPTION
**Pull request by whysage**
Fixes # 5712 added sleep tool

#### Who can review?

@vowelparrot

### CODE REVIEW

File: docs/modules/agents/tools/getting_started.md [hunk 1]
```diff
@@ -160,3 +160,9 @@ Below is a list of all supported tools and relevant information:
 - Notes: A connection to the OpenWeatherMap API (https://api.openweathermap.org), specifically the `/data/2.5/weather` endpoint.
 - Requires LLM: No
 - Extra Parameters: `openweathermap_api_key` (your API key to access this endpoint)
+
+**sleep**
```
Comment (vowelparrot) [inline docs/modules/agents/tools/getting_started.md:164]: Let's add to `langchain/agents/load_tools.py` for this one!

**Comment by vowelparrot** [comment]
Thanks for the PR! one main request - I'm also curious what you've found this useful for :)

File: docs/modules/agents/tools/getting_started.md (hunk 1 above)
Comment (whysage) [reply docs/modules/agents/tools/getting_started.md:164]: Thanks, done.

**Review by hwchase17 — APPROVED** [review]
looks reasonable to me - thanks!

**Pull request merged**